app/src/main/res/values/strings.xml

```diff
@@ -864,4 +864,6 @@
     <string name="fragment_enter_community_url_join_button_title">Join</string>
     <string name="new_conversation_dialog_back_button_content_description">Navigate Back</string>
     <string name="new_conversation_dialog_close_button_content_description">Close Dialog</string>
+    <string name="ErrorNotifier_migration">Database Upgrade Failed</string>
+    <string name="ErrorNotifier_migration_downgrade">Please contact support to report the error and then install an older version to continue using Session.</string>
```

Comment:
We might want to exclude the downgrade instructions, I'm not sure if people can downgrade through app installers or the app store, although it might be possible to force it via adb shell commands but otherwise it's pretty unsupported

Reply:
Huh, I could have sworn Google had an option in the Play Store where you could downgrade to an old version - might be misremembering (or it's been removed since I last saw it 😂)

I'll remove the `"and then..."` text and if we get any error reports we can just rollback on our side by releasing an update